src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clones = [];
```

Comment:
```suggestion
  const stateClones = [];
```

Not a big mistake, but use more descriptive names 🙌